sbncode/FlashMatch/CMakeLists.txt

```diff
@@ -31,6 +31,8 @@ art_make(
 
   larpandora_LArPandoraInterface
 
+  # caf_RecoUtils //uncomment for creating metrics
```

Comment:
I don't think that having to comment and recompile the code in order to be able to create the metrics is a great idea, it massively increases the maintenance burden if there are any upstream changes to the simulation or reconstruction. It would be preferable to have just behind a fcl parameter

Reply:
That's a good point. I've removed all of uncomments lines, now there's a new fcl specific to create metrics.